auth0/v3/management/email_templates.py

```diff
@@ -0,0 +1,65 @@
+from .rest import RestClient
+
+
+class EmailTemplates(object):
+
+    """Auth0 email templates endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/email-templates' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def create(self, body):
+        """Create a new email template.
+
+        Args: 
+           body (dict): Attributes for the new email template.
```

Comment:
Might be good to mention the required params. Saw your Jira note about this not being a convention here but could start with this endpoint? Not a show-stopper.

Reply:
Also, I guess that's why the repo owner linked to the documentation whenever a `body` was present in the function params.